packages/itwinui-css/src/slider/slider.scss

```diff
@@ -215,8 +223,11 @@ $tick-height: $iui-baseline; // 12px
 
   .iui-slider-min,
   .iui-slider-max {
-    margin-left: $iui-xxs;
+    margin-left: calc($iui-xxs + $iui-sm);
     text-align: center;
+    transform: translate(-50%, 0);
+    height: $iui-line-height;
```

Comment:
Is it okay to use `$iui-line-height` here? Or is it better to use some other hard coded value here?

Reply:
`$iui-line-height` doesn't exist anymore. I'm currently rebasing the main branch onto `v1` so I would say wait a bit and then update your branch to see what variables are available